acceptance/suites/puppet3_tests/010_hello_world/testtest.rb

```diff
@@ -0,0 +1,14 @@
+test_name "Testing Master/Agent backwards compatibility"
+
+# Agent running on the master is current, not legacy.
+legacy_agents = agents.reject { |agent| agent == master }
```

Comment:
The nomenclature here is a little strange, since it's not the fact that the agent machine isn't also the master that makes it "legacy", but the fact that it has puppet 3.x installed. I imagine this might be confusing for future readers of this code.